.github/workflows/publish.yml

```diff
@@ -0,0 +1,57 @@
+name: publish
+
+permissions:
+  contents: write
+
+on:
+  # schedule:
+  # - cron: "0 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v4
+
+    - name: Setup NodeJS
+      uses: actions/setup-node@v4
+      with:
+        node-version-file: "package.json"
+
+    - name: Setup pnpm
+      uses: pnpm/action-setup@v3
```

Comment:
PLease replace these actions with `jdx/mise-action`. The reference is [here ](https://github.com/ONLYOFFICE/api.onlyoffice.com/blob/3b8a5ce9fff563699764c0dca804ee394c5f2694/.github/workflows/publish.yml#L31).